### PR TITLE
changes in the event section for convoke 5.0

### PR DESCRIPTION
**Pull request by Akhil-Sharma30**
convoke changes

### CODE REVIEW

File: 2023/index.html [hunk 1]
```diff
@@ -2459,7 +2459,7 @@ <h1>
                                     <div class="description-inner">
                                         <h2><a href="https://clusterinnovationcentre.github.io/hashhacks/" target="_blank">Hash Hacks</a></h2>
                                         <p>A 24 hour Hackathon<br><a href="https://clusterinnovationcentre.github.io/hashhacks/"
-                                                target="_blank">&lt;register&gt;</a> <a
+                                                target="_blank">&lt;Coming Soon..&gt;</a> <a
```
Comment (Saransh-cpp) [inline 2023/index.html:2462]:
```suggestion
                                                target="_blank">&lt;Coming Soon&gt;</a> <a
```

File: 2023/index.html [hunk 2]
```diff
@@ -2501,7 +2501,7 @@ <h2><a href="https://www.facebook.com/events/501444797305451/"
                                         <h2><a href="https://www.facebook.com/events/919944925035947/"
                                                 target="_blank">Geeky Shark</a></h2>
                                         <p>The BizTech Quiz.<br><a href="https://forms.gle/zpNpLvszZAKUiSna8"
-                                                target="_blank">&lt;register&gt;</a> <a
+                                                target="_blank">&lt;Coming Soon...&gt;</a> <a
```
Comment (Saransh-cpp) [inline 2023/index.html:2504]:
```suggestion
                                                target="_blank">&lt;Coming Soon&gt;</a> <a
```

Comment (Saransh-cpp) [inline 2023/index.html:2504]: Comment out the link for now. The button should not work.

File: 2023/index.html [hunk 3]
```diff
@@ -2576,40 +2576,61 @@ <h2><a href="https://convoke-autonomi.github.io/robosoccer.html"
...
-                                        <div class="price">32<sup>seats</sup></div>
+                                        
                                         <!-- <div class="rating"><span class="icon icon-star"></span><span class="icon icon-star"></span><span class="icon icon-star"></span><span class="icon icon-star"></span></div> -->
                                     </div>
                                     <div class="description-inner">
                                         <h2><a href="https://www.facebook.com/events/748649172247607/"
                                                 target="_blank">CS:GO</a></h2>
                                         <p>Flicks and Strats.<br><a href="https://forms.gle/9qDTtyQtHLc2qEACA"
-                                                target="_blank">&lt;register&gt;</a> <a
+                                                target="_blank">&lt;Coming Soon...&gt;</a> <a
```
Comment (Saransh-cpp) [inline 2023/index.html:2633]:
```suggestion
                                                target="_blank">&lt;Coming Soon&gt;</a> <a
```

Comment (Saransh-cpp) [inline 2023/index.html:2633]: link

File: 2023/index.html [hunk 4]
```diff
@@ -2623,15 +2644,15 @@ <h2><a href="https://www.facebook.com/events/748649172247607/"
...
                                         <!-- <div class="rating"><span class="icon icon-star"></span><span class="icon icon-star"></span><span class="icon icon-star"></span><span class="icon icon-star"></span></div> -->
                                     </div>
                                     <div class="description-inner">
                                         <h2><a href="https://www.facebook.com/events/538309163642567/"
-                                                target="_blank">FIFA</a></h2>
+                                                target="_blank">FIFA 23</a></h2>
                                         <p>How the world plays soccer.<br><a
                                                 href="https://docs.google.com/forms/d/e/1FAIpQLScqpPdtG2v513WMQxTAgHwb10HNT-NGPG-VS4HyDeBVnLrzRQ/viewform?usp=pp_url"
-                                                target="_blank">&lt;register&gt;</a> <a
+                                                target="_blank">&lt;Coming Soon...&gt;</a> <a
```
Comment (Saransh-cpp) [inline 2023/index.html:2655]:
```suggestion
                                                target="_blank">&lt;Coming Soon&gt;</a> <a
```

Comment (Saransh-cpp) [inline 2023/index.html:2655]: link

File: 2023/index.html [hunk 5]
```diff
@@ -2623,15 +2644,15 @@ <h2><a href="https://www.facebook.com/events/748649172247607/"
...
                                                 href="mailto:gamingsocietycic@gmail.com?Subject=I%20need%20to%20know%20more%20about%20FIFA%20at%20CONVOKE%20"
```
Comment (Saransh-cpp) [inline 2023/index.html:2656]: neeche mail mei saransh0701@gmail.com

File: 2023/index.html [hunk 6]
```diff
@@ -2673,7 +2694,7 @@ <h2><a href="https://www.facebook.com/events/382235355797126/"
                                         <h2><a href="https://www.facebook.com/events/927958580915452/"
                                                 target="_blank">Tech talks</a></h2>
                                         <p>Experiences, tutorials, and much more!<br><a href="https://forms.gle/7QCCNHFh52LEA8Ed9"
-                                                target="_blank">&lt;register&gt;</a> <a
+                                                target="_blank">&lt;Coming Soon...&gt;</a> <a
```
Comment (Saransh-cpp) [inline 2023/index.html:2697]:
```suggestion
                                                target="_blank">&lt;Coming Soon&gt;</a> <a
```

Comment (Saransh-cpp) [inline 2023/index.html:2697]: link and mail

File: 2023/index.html [hunk 7]
```diff
@@ -2701,17 +2722,17 @@ <h2><a href="https://www.facebook.com/events/919944925035947/"
                                     </div>
                                 </div>
 
-                            </div>
+                            </div> -->
                         </li>
```
Comment (Saransh-cpp) [inline 2023/index.html:2726]: Should this be commented?

File: 2023/index.html [hunk 8]
```diff
@@ -2576,40 +2576,61 @@ <h2><a href="https://convoke-autonomi.github.io/robosoccer.html"
...
+                                        <div class="price">150<sup>seats</sup></div>
+                                        <!-- <div class="rating"><span class="icon icon-star"></span><span class="icon icon-star"></span><span class="icon icon-star"></span><span class="icon icon-star"></span></div> -->
+                                     </div>
+                                    <div class="description-inner">
+                                        <h2><a href="https://www.facebook.com/events/387953888823650/"
+                                                target="_blank">Treasure hunt</a></h2>
+                                        <p>The future is here.<br><a href="https://forms.gle/MSRkyxgGyfZtm9b89"
+                                                target="_blank">&lt;Coming Soon...&gt;</a> <a
+                                                href="mailto:rudrankriyam@gmail.com?cc=convoke4.0@gmail.com&Subject=I%20need%20to%20know%20more%20about%20SwiftUI%20Workshop%20"
+                                                target="_blank">&lt;queries?&gt;</a></p>
```
Comment (Saransh-cpp) [inline 2023/index.html:2614]: Bohot mistakes, fix.

**Review by Saransh-cpp — CHANGES_REQUESTED** [review]

**Comment by Saransh-cpp** [comment]
Images should have the same dimensions

**Comment by Saransh-cpp** [comment]
Clean up the code comments. For example - 

image

A random comment sign on the website.

**Comment by Saransh-cpp** [comment]
Also, update your main branch.

**Comment by Akhil-Sharma30** [comment]
image

@Saransh-cpp made the changes and fixed the `Tech Talk` image.

**Comment by Akhil-Sharma30** [comment]
> image

@Saransh-cpp made the changes and fixed the `Tech Talk` image.

**Review by Saransh-cpp — APPROVED** [review]
Nice, thanks!

**Pull request merged**